pages/linux/zramctl.md

```diff
@@ -0,0 +1,23 @@
+# zramctl
+
+> Setup and control zram devices.
+
+- Check if zram is enabled:
+
+`lsmod | grep -i zram`
+
+- Enable zram with 2 devices:
+
+`sudo modprobe zram num_devices={{2}}`
+
+- Initialise the next free zram device to a 2GB virtual drive:
+
+`sudo zramctl --find --streams {{4}} --size {{2GB}} --algorithm lz4`
```

Comment:
Yeah, that probably ought to be added to the description. It specifies the number of compression streams that should be attached to the device. I interpret that to be the number of concurrent reads/writes it can handle at once. Either way, I assume you'd want to set it to the number of cpus you have for best performance.